bigbluebutton-html5/imports/ui/components/video-provider/service.js

```diff
@@ -918,54 +953,11 @@ class VideoService {
     return constraints;
   }
 
-  applyCameraProfile (peer, profileId) {
-    const profile = CAMERA_PROFILES.find((targetProfile) => targetProfile.id === profileId);
-
-    // When this should be skipped:
```

Comment:
The replacement portion seems more concise. Do we lose any functionality though? I am not spotting some of the proofchecks. Do we no longer need to log `video_provider_constraintchange_failed` ?

Reply:
Some functions from video-provider service have been replaced by similar ones in graphql service file (see https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.x-release/bigbluebutton-html5/imports/ui/components/video-provider/video-provider-graphql/service.ts#L422)